### PR TITLE
Added blog entry for the minutes of meetings

### LINKED ISSUE #12: Add blog entry for minutes of meetings
**Describe the bug/feature**

Blog entry for the minutes of meetings have to be added. The meeting was held a few days back with the officials of our parent club.

**To Reproduce(for bug)**

**Screenshots(for bug)**

**Device Details(for bug)**

**Other Comments**

The content is in the file given below.
[board_meeting.txt](https://github.com/rtrclubiitj/rtrclubiitj.github.io/files/3506463/board_meeting.txt)

@Saptashrungi Please do the needful, probably before the next meeting.

### PR DESCRIPTION
**Pull request by Saptashrungi**
Blog

#### Description

Fixes #12 
Added md file for minutes of board meetings. The blog however is incomplete in YAML in case of image,author .

### CODE REVIEW

**Comment by Saptashrungi** [comment]
ping @czgdp1807

File: _posts/2019-08-18-post_8.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿---
+layout: post
+title:  "Minutes Of Meetings"
+date:   2019-08-12
+author: ""
+excerpt: "Rotaract club organised a productive meeting."
```
Comment (czgdp1807) [inline _posts/2019-08-18-post_8.md]: Use `Progressive minds focus on ideas.` as excerpt.

File: _posts/2019-08-18-post_8.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+image: ""
+is_pinned:
```
Comment (czgdp1807) [inline _posts/2019-08-18-post_8.md]: Should be `false`.

**Comment by czgdp1807** [comment]
A new PR? What was wrong with the previous one?

File: _posts/2019-08-18-post_8.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿---
+layout: post
+title:  "Minutes Of Meetings"
+date:   2019-08-12
```
Comment (czgdp1807) [inline _posts/2019-08-18-post_8.md]: Is the date of the meeting?

**Pull request closed without merging**